build.rs

```diff
@@ -1,103 +1,53 @@
 use bindgen;
-#[cfg(all(
-    not(feature = "pkg_config"),
-    feature = "bundled_proj",
-    not(feature = "nobuild")
-))]
 use cmake;
-#[cfg(all(
-    not(feature = "pkg_config"),
-    feature = "bundled_proj",
-    not(feature = "nobuild")
-))]
 use flate2::read::GzDecoder;
-#[cfg(all(
-    not(feature = "pkg_config"),
-    feature = "bundled_proj",
-    not(feature = "nobuild")
-))]
 use std::fs::File;
 
-#[cfg(all(
-    feature = "pkg_config",
-    not(feature = "bundled_proj"),
-    not(feature = "nobuild")
-))]
 use pkg_config;
 use std::env;
 use std::path::PathBuf;
-#[cfg(all(
-    not(feature = "pkg_config"),
-    feature = "bundled_proj",
-    not(feature = "nobuild")
-))]
 use tar::Archive;
 
-#[cfg(all(
-    feature = "pkg_config",
-    not(feature = "bundled_proj"),
-    not(feature = "nobuild")
-))]
 const MINIMUM_PROJ_VERSION: &str = "7.1.0";
 
 #[cfg(feature = "nobuild")]
 fn main() {} // Skip the build script on docs.rs
 
-// We sometimes need additional search paths, which we get using pkg-config
-#[cfg(all(
-    feature = "pkg_config",
-    not(feature = "nobuild"),
-    not(feature = "bundled_proj")
-))]
-fn main() {
-    let pk = pkg_config::Config::new()
+#[cfg(not(feature = "nobuild"))]
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let include_path = if cfg!(feature = "bundled_proj") {
+        println!("feature flags specified source build");
```

Comment:
Should all the "info" messages print to stderr (`eprintln!`) ?

Reply:
cargo only looks at stdout lines that begin with "cargo:" - so this won't break anything, but I've switched it to eprintln. I'm fine with it either way.